### PR TITLE
fix: update chrome to 128 firefox to 130 inside Cypress tests in CircleCI

### PR DESCRIPTION
**Pull request by AtofStryker**
- Closes N/A

### Additional details

As a prerequisite for doing the WebDriver BiDi work for Firefox, we needed to update to a minimum of Firefox 129 to get mostly complete BiDi support. This knocks out the chrome update as well to be on chrome latest

The main changes were some screenshot height differences, which has been a problem for chrome in the past. We are now seeing it with Firefox as well, though the height for the default parameters is only 1 pixel off. I added some code to mitigate this (a bit hacky, but gets the job done).

There was also a regression introduced with Firefox 129 that incorrectly mutated the checkbox prop on an event if a click event on a checkbox was sent in 129 or up (hence the fix tag)

### Steps to test

Test against CI or run the `type_ events` cypress test with/without the fix to reproduce as long as you are on Firefox 129 and above.

### PR Tasks

- [x] Have tests been added/updated?
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

File: system-tests/__snapshots__/vite_dev_server_fresh_spec.ts.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ We dynamically generated a new test to display this failure.
 
   (Screenshots)
 
-  -  /XXX/XXX/XXX/cypress/screenshots/AppCompilationError.cy.jsx/An uncaught error wa     (1280x633)
+  -  /XXX/XXX/XXX/cypress/screenshots/AppCompilationError.cy.jsx/An uncaught error wa     (1280x581)
```
Comment (jennifer-shehane) [inline system-tests/__snapshots__/vite_dev_server_fresh_spec.ts.js:85]: Are these tests actually testing screenshot behavior? Can we just turn the screenshots off here?

(Not suggesting this optimization be a part of this PR, I could maybe follow up with a PR for this if so)

File: system-tests/lib/normalizeStdout.ts [hunk 2]
```diff
@@ -178,6 +178,25 @@ export const normalizeStdout = function (str: string, options: any = {}) {
     str = str.split('\n').filter((line) => !line.includes(wdsFailedMsg)).join('\n')
   }
 
+  // in Firefox 130, height dimensions are off by 1 pixel in CI, so we need to fix the offset to match common snapshots
+  if (options.browser === 'firefox' && process.env.CI) {
+    const dimentionRegex = new RegExp(/(\((?<width>\d+)x(?<height>\d+)\))/g)
```
Comment (jennifer-shehane) [inline system-tests/lib/normalizeStdout.ts]: typo

```suggestion
    const dimensionRegex = new RegExp(/(\((?<width>\d+)x(?<height>\d+)\))/g)
```

File: system-tests/lib/normalizeStdout.ts [hunk 3]
```diff
@@ -178,6 +178,25 @@ export const normalizeStdout = function (str: string, options: any = {}) {
...
+
+    const matches = dimentionRegex.exec(str)
```
Comment (jennifer-shehane) [inline system-tests/lib/normalizeStdout.ts]:
```suggestion
    const matches = dimensionRegex.exec(str)
```

**Review by jennifer-shehane — DISMISSED** [review]

File: system-tests/__snapshots__/vite_dev_server_fresh_spec.ts.js (hunk 1 above)
Comment (AtofStryker) [reply system-tests/__snapshots__/vite_dev_server_fresh_spec.ts.js:85]: I don't think they are. Just looks like the screenshot taken from a given failure

Comment (jennifer-shehane) [reply system-tests/__snapshots__/vite_dev_server_fresh_spec.ts.js:85]: K, I'll take a look at this.

**Review by jennifer-shehane — APPROVED** [review]

**Review by ryanthemanuel — APPROVED** [review]

**Pull request merged**